context/app/templates/special-pages/ccf-eui.html

```diff
@@ -1,23 +1,24 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
 
 <head>
-  <meta charset="utf-8">
-  <title>HuBMAP CCF Exploration User Interface (CCF-EUI)</title>
-  <base href="https://cdn.jsdelivr.net/gh/hubmapconsortium/ccf-ui@3/">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
-  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500&amp;display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons|Material+Icons+Outlined" rel="stylesheet">
-  <link rel="stylesheet" href="styles.css">
+  <meta charset="utf-8" />
+  <title>HuBMAP HRA Exploration User Interface</title>
+  <base href="https://cdn.humanatlas.io/ui/ccf-eui/" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="icon" type="image/png" href="/favicon.png" />
```

Comment:
```suggestion
  <link rel="icon" type="image/x-icon" href="/favicon.ico" />
```

We have a `.ico` icon, not a `.png` icon: https://github.com/hubmapconsortium/portal-ui/blob/main/context/app/static/favicon.ico

Reply:
This actually refers to the favicon at https://cdn.humanatlas.io/ui/ccf-eui/ as it is using https://cdn.humanatlas.io/ui/ccf-eui/ as the base-href (see line 7)

Reply:
Ah, right, good call - I missed that detail.

I think it'll be all set to merge if the CCFOrganInfo component is up to date.

Reply:
Ok, sounds good. Like I said with the organ info, I'd like to keep the `?token=` as is since it will help tremendously with caching/responsiveness.

Reply:
Yes - my question is about the values here: https://github.com/hubmapconsortium/portal-ui/blob/update-hra/context/app/static/js/components/HRA/CCFOrganInfo.tsx#L29-L30

Reply:
Oh! Thank you, yes, I'll get those updated.

Reply:
Ok, pushed it!